identity.proto

```diff
@@ -0,0 +1,13 @@
+syntax = "proto3";
+package dogma.interop.v1;
+
+option go_package = "github.com/dogmatiq/interopspec";
+
+// Identity is a protocol buffers representation of the configkit.Identity type.
+message Identity {
```

Comment:
The `envelopespec` package should use this type instead of its own `Identity` type.

Reply:
Good catch, fixing now...

Reply:
This should be fixed with https://github.com/dogmatiq/interopspec/pull/2/commits/d687f462d657d51e3279f2b90054641ed1194466